.github/ISSUE_TEMPLATE/new-event-fellowship-round.md

```diff
@@ -0,0 +1,60 @@
+---
+name: New Round for OBF Event Fellowship Application
+about: 'This template is for the OBF Event Fellowship chair to manage a new application round'
+
+---
+
+**OBF Event Fellowship Round:**
+
+- [ ] Round-1: 15 May
+- [ ] Round-2: 15 September
+- [ ] Round-3: 15 January
+
+**Before opening the call for application (minimum 6 weeks in advance)**
+
+*7-8 weeks before the deadline:*
+- [ ] Update the application form with the correct date and application round-specific information (such as BOSC participation)
+- [ ] Create a blog post for the announcement
+- [ ] Get the board to review the blog, and when the majority agrees, publish the blog
+- [ ] Cross-post announcements on Twitter, Slack and newsletter (OBF or other projects)
+- [ ] If possible, host a webinar to share tips and suggestions with the potential applicants
+
+*1-2 weeks before the deadline:*
+- [ ] Amplify the announcement
+- [ ] Create a folder for the specific round (within the dedicated directory for event fund) in the OBF-board shared google drive (round-specific shared folder)
+- [ ] Within that folder, create a copy of the rubric template to share with the board when the review begins
+- [ ] Double check if any application has come through that may need to follow up for any missing detail
+
+**After the deadline**
+
```

Comment:
```suggestion

```
- [ ] Change the deadline listed at the beginning of the application form (currently https://docs.google.com/forms/d/e/1FAIpQLSc1DFilEFV0R0hzsLGI7JkE3UHQgqQQnNXDztvqieE2TmhOMA/viewform) to the next deadline.

Reply:
I guess December 1 would be too close for this round. Changed to April 1 2023